src/main/kotlin/it/pagopa/wallet/util/LogoUtils.kt

```diff
@@ -0,0 +1,25 @@
+package it.pagopa.wallet.util
+
+import it.pagopa.wallet.domain.wallets.Wallet
+import it.pagopa.wallet.domain.wallets.details.CardDetails
+import it.pagopa.wallet.domain.wallets.details.PayPalDetails
+import java.net.URI
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+@Component
+class LogoUtils(@Autowired val logoMapping: Map<String, URI>) {
```

Comment:
Don't love the name but can't think of a much better one 🤔
Maybe `LogoResources`? Idk, don't love that either, still posting a comment to spark a discussion

Reply:
What about `WalletsLogoUtils`?

Reply:
Refactored to WalletUtils with https://github.com/pagopa/pagopa-wallet-service/pull/100/commits/98478a93356237d33d775a85c9b25ec4741cc5f4